README.md

```diff
@@ -54,6 +54,66 @@ PHP CodeSniffer can be configured by adding a `phpcs.xml` file to the source cod
 
 Currently, the tool supports this configuration file except from one feature - setting installed_paths: `<config name="installed_paths" />`
 
+## Add new Codesniffer plugin
+
+To add a new plugin to Codesniffer, you should:
+
+1.  Add the plugin dependency to `requirements` inside `composer.json`. This depedency must be available on [packagist](https://packagist.org/)
```

Comment:
The correct JSON node seems to be `require` (https://github.com/codacy/codacy-codesniffer/blob/master/composer.json):

```suggestion
1.  Add the plugin dependency to `require` inside `composer.json`. This depedency must be available on [Packagist](https://packagist.org/)
```